src/pages/[platform]/build-a-backend/functions/add-lambda-layers/index.mdx

```diff
@@ -0,0 +1,60 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Lambda Layers',
+  description:
+    'Learn how to add layers to your function',
+  platforms: [
+    'android',
+    'angular',
+    'flutter',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+  ]
+};
+
+export function getStaticPaths() {
+  return getCustomStaticPath(meta.platforms);
+}
+
+export function getStaticProps() {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+Amplify offers the ability to add layers to your Functions which contain your library dependencies. To get started, specify the `layers` property in `defineFunction`:
+
+```ts title="amplify/functions/my-function/resource.ts"
+import { defineFunction } from "@aws-amplify/backend";
+
+export const myFunction = defineFunction({
+  name: "my-function",
+  layers: {
+   "@aws-lambda-powertools/logger":
+      "arn:aws:lambda:us-east-1:094274105915:layer:AWSLambdaPowertoolsTypeScriptV2:12",
+  },
+});
+```
+
+The Lambda layer is represented by an object of key/value pairs where the key is the module name that is exported from your layer and the value is the ARN of the layer. The key (module name) is used to externalize the module dependency so it doesn't get bundled with your lambda function. A maximum of 5 layers can be attached to a function, and they must be in the same region as the function.
+
+then use the locally installed module in the function handler:
+```ts title="amplify/functions/my-function/handler.ts"
+import { Logger } from "@aws-lambda-powertools/logger";
+
+const logger = new Logger({ serviceName: "serverlessAirline" });
+
+export const handler = async (): Promise<void> => {
```

Comment:
```suggestion
export const handler = async (): Promise<void> => {
```

can we show this with one of the generic `Handler` lambda types?

Reply:
updated the snippet